### PR TITLE
Implement quickfix for wrong primitive literal

### PR DESCRIPTION
**Pull request by cypressious**
Fixes: KT-12251

Not sure if I should disable the other quickfix which adds a call to `.toX()`

### CODE REVIEW

**Comment by cypressious** [comment]
Rewrote the fix and disabled the `.toX()` fix hen the new is active. Also added some tests.

File: idea/src/org/jetbrains/kotlin/idea/quickfix/WrongPrimitiveLiteralFix.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            else if ('.' !in this) {
+                append(".0")
+            }
+        }
+        else {
+            if (constValue is Float || constValue is Double) {
+                append(constValue.toLong())
+            }
+            else {
+                append(element.text.takeWhile { it != 'l' && it != 'L' })
```
Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/quickfix/WrongPrimitiveLiteralFix.kt]: I'd rather use `element.trimEnd('l', 'L')`

**Comment by yole** [comment]
Looks good except for the minor style issue.

**Comment by cypressious** [comment]
@yole fixed the style issue

**Pull request merged**